docs/proxy/google_serp_proxy/examples.md

```diff
@@ -0,0 +1,512 @@
+---
+title: Examples
+description: Learn how to connect to Google SERP proxies from your applications with Node.js (axios and got), Python 2 and 3 and PHP using code examples.
+paths:
+    - proxy/google-serp-proxy/nodejs-examples
+    - proxy/google-serp-proxy/python-examples
+    - proxy/google-serp-proxy/php-examples
+    - proxy/google-serp-proxy/examples
+---
+<!-- Watch out! This file is hard to read because our own "marked-tabs" language fences aren't syntax-highlighted -->
+
+# Connect to Google SERP proxies
+
+This page contains code examples for connecting to [Google SERP proxies]({{@link proxy/google_serp_proxy.md}}) using [Apify Proxy](https://apify.com/proxy).
+
+If you are building your own Apify [actor]({{@link actors.md}}), below are [examples](#using-the-apify-sdk) specific to the [Apify SDK](https://sdk.apify.com).
+
+See the [connection settings]({{@link proxy/connection_settings.md}}) page for connection parameters.
+
+## [](#using-standard-libraries-and-languages) Using standard libraries and languages
+
+You can find your proxy password on the [Proxy page](https://my.apify.com/proxy) of the Apify app.
+
+> The `username` field is **not** your Apify username.<br/>
+> Instead, you specify proxy settings (e.g. `groups-GOOGLE_SERP`).<br/>
+> Use `groups-GOOGLE_SERP` to use proxies from all available countries.
+
+For examples using [PHP](https://www.php.net/), you need to have the [cURL](https://www.php.net/manual/en/book.curl.php) extension enabled in your PHP installation. See [installation instructions](https://www.php.net/manual/en/curl.installation.php) for more information.
+
+Examples in [Python 2](https://www.python.org/download/releases/2.0/) use the [six](https://pypi.org/project/six/) library. Run `pip install six` to enable it.
+
+### [](#html-from-search-results) HTML from search results
+
+Get the HTML of search results for the keyword **wikipedia** from the USA (**google.com**).
+
+Select this option by setting the `username` parameter to `groups-GOOGLE_SERP`. Add the item you want to search to the `query` variable.
+
+```marked-tabs
+<marked-tab header="Node.js (axios)" lang="javascript">
+const HttpsProxyAgent = require("https-proxy-agent");
+const axios = require("axios");
+
+const httpsAgent = new HttpsProxyAgent({
+    host: "proxy.apify.com",
+    port: "8000",
+    // Replace <YOUR_PROXY_PASSWORD> below with your password
+    // found at https://my.apify.com/proxy
+    auth: "groups-GOOGLE_SERP:<YOUR_PROXY_PASSWORD>"
+});
+
+axiosHttpsAgent = axios.create({ httpsAgent });
+
+// Encode your query as a URI parameter
+const query = `q=${encodeURIComponent('wikipedia')}`;
+
+async function useProxy() {
+    const response = await axiosHttpsAgent.get(`http://www.google.com/search?${query}`);
```

Comment:
I would use `axios`'s `params` option instead of encoding the query manually and then constructing the URL from strings.